app/src/main/java/de/danoeh/antennapod/discovery/ItunesTopListLoader.java

```diff
@@ -22,18 +24,19 @@
 
 public class ItunesTopListLoader {
     private final Context context;
+    String LOG = "ITunesTopListLoader";
```

Comment:
In other classes, we call this field TAG (and it's private static final)

Reply:
Could you also please make it `private static final`? That way, the compiler can even inline the string.